src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -107,62 +113,63 @@ def move_by(self, increment):
         self.logger.debug(f"Moved by {increment} encoder units. Current position is {new_pos}")
         return new_pos
```

Comment:
In order for things to be consistent, this should probably be `self.position`. Also, with the Birger class, `move_by` commands actually return the encoder increment rather than the actual position. Although that does seem inconsistent with the abstract focuser class. That might lead to some serious errors if there are a mix of astromechs and birgers as we have on Huntsman.

Reply:
I'm going to do a separate small PR.

Reply:
https://github.com/panoptes/POCS/pull/1126